src/migrations/1607282312831-AddNotesColumnToVolunteersTable.ts

```diff
@@ -0,0 +1,18 @@
+import { Table, MigrationInterface, QueryRunner, TableColumn } from "typeorm";
+
+export class AddNotesColumnToVolunteersTable1607282312831 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner) {
+    return queryRunner.addColumn(
+      new Table({ name: "Volunteers" }),
+      new TableColumn({
+        name: "notes",
+        type: "varchar",
```

Comment:
me fijaria si existe el typetext o algo que no tiene limite.

Reply:
Según lo que veo parece ser lo mismo ([link](https://www.postgresqltutorial.com/postgresql-char-varchar-text/))

If you do not specify the n integer for the VARCHAR data type, it behaves like the TEXT datatype. The performance of the VARCHAR (without the size n) and TEXT are the same.

Reply:
ah claro, tiene sentido, genial ahi resuelvo el comment